mainwindow.cc

```diff
@@ -3241,11 +3241,11 @@ void MainWindow::setAutostart(bool autostart)
     }
     reg.sync();
 #else
-    // this is for KDE
+    // this is for Linux
     QString app_fname = QFileInfo(QCoreApplication::applicationFilePath()).baseName();
```

Comment:
This variable is now unused and can be removed.

Reply:
"Now" includes all GNU/Linux distributions, for which current Goldendict version is compilable?

Reply:
Well, this is a local variable and you removed its single use on the line below. So I'm pretty sure it's note used anymore on any platform :)